tasks/patch_wordpress.js

```diff
@@ -280,4 +285,76 @@ module.exports = function(grunt) {
 
 	})
 
+	grunt.registerTask( 'upload_patch_to_wordpress_trac', 'Upload the current diff of your develop-wordpress directory to WordPress trac', function( ticketNumber ) {
+		var done = this.async()
+		var options = this.options(defaults)
+
+		grunt.log.debug( 'ticketNumber: ' + ticketNumber )
+		grunt.log.debug( 'options: ' + JSON.stringify( options ) )
+
+		var requestTicketNumber = function() {
+			rl.question('Enter the ticket number: ', function(_ticketNumber) {
```

Comment:
In the `patch_wordpress` task, we take the ticket number in as a part of the cli call?  What do you think of doing that here to stay consistent?